drops/esa_ubuntu_k3_single.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Title": "Create ESA instance on Single Node Ubuntu K3s",
+    "Summary": "This Jumpstart Drop provides Kubernetes files to create an ESA Cache Volume instance on an install on Ubuntu with K3s.",
+    "Description": "This project provides example Kubernetes files to create an ESA Cache Volume backed by your storage account. See README.md for details on editing and executing these kubernetes files.",
+    "Cover": "",
+    "Authors": [
+        {
+            "Name": "Anthony Joint",
+            "Link": "https://www.linkedin.com/in/anthonyjoint/"
+        }
+   ],
+    "Source": "https://github.com/esa_jumpstart_drops/esa_ubuntu_k3_single",
```

Comment:
Source does not exsits

Reply:
Source is in this PR, so it won't exist until the PR merges.

Reply:
Canary should be a good place to validate.